src/docker.rs

```diff
@@ -163,10 +163,13 @@ pub fn run(target: &Target,
 
     if atty::is(Stream::Stdout) && atty::is(Stream::Stderr) {
         docker.arg("-t");
+        if atty::is(Stream::Stdin) {
+            docker.arg("-i");
```

Comment:
I only meant moving this line, so make the arg above `-it`.

Reply:
I guess we should replace `atty::is(Stream::Stdout) && atty::is(Stream::Stderr)` with just `atty::is(Stream::Stdin)`.

Reply:
Oh whoops. Sure.

Reply:
Not checking for stdout and stderr anymore doesn't cause anymore problems?

Reply:
Not sure, maybe we should swap the order instead.

If `stdin` is a TTY, use `-i`, and additionally if both `stdout` and `stderr` are a TTY, also use `-t`.

Reply:
I'll change it to that, as that sounds safer than not checking at all.